portal/views/portal.py

```diff
@@ -707,8 +707,8 @@ def settings():
 @oauth.require_oauth()
 def config_settings(config_key):
     key = config_key.upper()
-    # Only handing out LifeRay keys at this time
-    if key.startswith('LR_'):
+    # check key existence in application config
+    if key in current_app.config:
```

Comment:
This is a security vulnerability since we store some secrets as config values :) (`SECRET_KEY`, `SQLALCHEMY_DATABASE_URI`, etc)

Is there a pattern in the config values you need so we can use something like `LR_` to allow access to some? 
If it's only a handful (less than ~10) it's probably fine to list them all.

Reply:
@ivan Ivan, didn't think of that.  thank you for catching this!  I added a check for "consent_" prefix config variables (e.g. CONSENT_WITH_TOP_LEVEL_ORG, COSENT__PERMISSIBLE_ROLES) that are accessed by the front end currently.  There may be others but those are the ones needed by the accountCreation.js.